bin/nanocl/src/commands/stats.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+  config::CliConfig,
+  models::{ProcessStatsRow, StatsArg},
+  utils,
+};
+use futures::channel::mpsc;
+use futures::StreamExt;
+use nanocl_error::io::IoResult;
+use std::collections::HashMap;
+
+pub async fn exec_stats(
+  cli_config: &CliConfig,
+  stats_arg: &StatsArg,
+) -> IoResult<()> {
+  let client = cli_config.client.clone();
+  let terminal = dialoguer::console::Term::stdout();
+  let (tx_stat, mut recv_stat) = mpsc::unbounded();
+  {
+    let proc_list = stats_arg.process_list.clone();
+    let client = client.clone();
+    ntex::rt::spawn(async move {
+      let fut = proc_list.iter().map(|x| async {
+        let Ok(mut stream) = client.stats_processes_by_name(x).await else {
+          return;
+        };
+        while let Some(stat_res) = stream.next().await {
+          if let Ok(stat) = stat_res {
+            let _ = tx_stat.unbounded_send(stat);
+          }
+        }
+      });
+      futures::future::join_all(fut).await;
+    });
+  }
+  let mut proc_map = HashMap::new();
```

Comment:
We don't need to use an hashmap a vector is enough there, the reason i do it with the cargo command is to have them ordered by cargoes.

Reply:
I think we need hashmap because it will create multiple entries for single process without it.

Reply:
Ah yeah i see what you mean, fair point